src/logging.c

```diff
@@ -17,10 +17,20 @@ extern "C"
 {
 #endif
 
+#include <string.h>
+
 #include "rcutils/allocator.h"
+#include "rcutils/find.h"
+#include "rcutils/get_env.h"
 #include "rcutils/logging.h"
+#include "rcutils/snprintf.h"
+
+#define RCUTILS_LOGGING_MAX_OUTPUT_FORMAT_LEN 1024
```

Comment:
This constant seems to be used for two purposes: for the max length of the environment variable as well as for the max length of every formatted message.

I would argue it is only necessary for the second case. Why do we need to limit the length of the environment variable (which is already limited by the operating system)?

Reply:
I think there's been a misinterpretation on the second use of the constant: there isn't currently any limit to the length of a formatted message. When `RCUTILS_LOGGING_MAX_OUTPUT_FORMAT_LEN` is used [here](https://github.com/ros2/rcutils/pull/53/files#diff-f909d0b1a298531f76132943dc55f86aR251) it's to put a conservative limit on the length of an individual token (not its expansion).

To your point about limiting the length of the environment variable, since the contents of the environment variable [needs to be copied in](https://github.com/ros2/rcutils/pull/53/files#diff-f909d0b1a298531f76132943dc55f86aR53), I set a limit so we could declare the buffer statically to avoid the need to dynamically allocate in `rcutils_logging_initialize`, since there's no logging fini at the moment. Provided this reasoning makes sense, how about I just increase the limit to 2048 to match the [max returned env var length on windows](https://github.com/ros2/rcutils/blob/450002515fe45a64e57e90257fc2f9ac68c1f6de/src/get_env.c#L26)?

Reply:
Thanks for clarifying.

> how about I just increase the limit to 2048 to match the max returned env var length on windows?

Sounds good to me.

Reply:
OK, 41e93ab3ff769f4aeb42cc6d0f5364a8e3b66590